resources/assets/js/app.js

```diff
@@ -6,17 +6,3 @@
  */
 
 require('./bootstrap');
-
-window.Vue = require('vue');
-
-/**
- * Next, we will create a fresh Vue application instance and attach it to
- * the page. Then, you may begin adding components to this application
- * or customize the JavaScript scaffolding to fit your unique needs.
- */
-
-Vue.component('example-component', require('./components/ExampleComponent.vue'));
```

Comment:
Why remove Vue? It is quite a nice framework that could really simplify your front-end logic.